student.rb

```diff
@@ -1,12 +1,17 @@
 require_relative 'person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, parent_permission: parent_permission, name: name)
-    @classroom = classroom
+  attr_accessor :classrrom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
   end
 
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def add_student
+    classroom.add_student(self) unless classroom.students.include?(self)
+  end
```

Comment:
- [ ] Good job trying to fix change(s) mentioned by the previous reviewer. However, as you are supposed to add a method that serves a student belongs to a classroom, kindly ensure that you update it to something similar to this one:
```ruby
  attr_reader :classroom

 def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, parent_permission: parent_permission, name: name)
    @classroom = classroom
 end
  .
  .
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```